dev-haskell/yesod-auth-hashdb/yesod-auth-hashdb-1.7.1.7.ebuild

```diff
@@ -1,16 +1,15 @@
-# Copyright 1999-2021 Gentoo Authors
+# Copyright 1999-2024 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=8
 
-# ebuild generated by hackport 0.6.7.9999
+# ebuild generated by hackport 0.8.5.1
```

Comment:
This commit doesn't bump the version, it just re-generates the ebuild with a newer version of hackport.